proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -35,6 +35,35 @@ message BreakingChangePolicy {
   BreakingChangeCategory category = 3;
 }
 
+message UniquenessPolicy {
+  optional bool enabled = 1;
+}
+
+message UniquenessState {
+  bool has_collisions = 1;
+  repeated Collision collisions = 2;
```

Comment:
we could probably reduce this to just the `repeated Collisions` which would make `has_collisions` == `len(collisions) != 0`?